context/UserContext.js

```diff
@@ -22,6 +24,14 @@ function reducer(state, action) {
             return { ...state, userEvents: action.value };
         case actions.SET_CLUBS:
             return { ...state, userClubs: action.value };
+        case actions.REMOVE_EVENTS:
+            let eventList = JSON.stringify(action.value).split('\"');
+            let eventName = eventList[eventList.length - 2];
+            return { userEvents: state.userEvents.filter(currEvent => currEvent != eventName) };
+        case actions.REMOVE_CLUBS:
+            let clubList = JSON.stringify(action.value).split('\"');
+            let clubName = clubList[clubList.length - 2];
+            return { userClubs: state.userClubs.filter(currClubs => currClubs != clubName) };
```

Comment:
I feel like there's a better way to filtering out the results than using the `split()`, but we can just fix it during our OA. For now, it will do since it works fine.